test/unit/core.js

```diff
@@ -791,6 +791,22 @@ QUnit.test( "jQuery.map", function( assert ) {
 		return k % 2 ? k : [ k, k, k ];
 	} );
 	assert.equal( result.join( "" ), "00012223", "Array results flattened (#2616)" );
+
+	result = jQuery.map( [ [ [ 1, 2 ], 3 ], 4 ], function( v, k ) {
+		return v;
+	} );
+	assert.equal( result.length, 3, "Array flatten only one level down" );
+	assert.ok( Array.isArray( result[ 0 ] ), "Array flatten only one level down" );
+
+
+	if ( Array.prototype.flat ) {
```

Comment:
Please add a support comment similar to the one in `src/var/flat.js`:
```js
// Support: IE 11+, Edge 18+
// Skip the test in browsers without Array#flat.
```

Reply:
Added :smiley: